docs/GettingStartedDocs/Windows_using_oe_sdk.md

```diff
@@ -0,0 +1,67 @@
+# Using the Open Enclave SDK on Windows
+
+This document provides a brief overview of how to start exploring the Open Enclave SDK
+once you have it installed.
+
+## Open Enclave SDK Layout
+
+On Windows, if you installed the SDK using the NuGet package, it is by default installed to `%userprofile%\.nuget\packages`.
+If you built the SDK from source and installed it, the SDK is installed to the location specified by CMAKE_INSTALL_PREFIX as described [here](Contributors/WindowsInstallInfo.md#basic-install-on-windows)
+
+It contains the following subfolders:
+
+| Path                         | Description                     |
+|------------------------------|---------------------------------|
+| bin                          | Developer tools such as oedebugrt.dll for debugging and oesign for signing your enclaves. |
```

Comment:
I just realized we don't have links in the SDK usage doc flow that talks about copying debugrt.dll to the target app folder for debugging, or setting up debugging. @radhikaj & @anakrish, is that underway?

Reply:
Yes, next task I will work on.

Reply:
@CodeMonkeyLeet I fixed this. Thanks.